client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoResultSet.java

```diff
@@ -0,0 +1,262 @@
+
+                        @Override
+                        public boolean isClientAborted()
+                        {
+                            return false;
+                        }
+
+                        @Override
+                        public boolean isClientError()
+                        {
+                            return false;
+                        }
+
+                        @Override
+                        public boolean isFinished()
+                        {
+                            return false;
+                        }
+
+                        @Override
+                        public StatementStats getStats()
+                        {
+                            return null;
+                        }
+
+                        @Override
+                        public QueryStatusInfo currentStatusInfo()
+                        {
+                            return null;
+                        }
+
+                        @Override
+                        public QueryData currentData()
+                        {
+                            return null;
+                        }
+
+                        @Override
+                        public QueryStatusInfo finalStatusInfo()
+                        {
+                            return null;
+                        }
+
+                        @Override
+                        public Optional<String> getSetCatalog()
+                        {
+                            return Optional.empty();
+                        }
+
+                        @Override
+                        public Optional<String> getSetSchema()
+                        {
+                            return Optional.empty();
+                        }
+
+                        @Override
+                        public Optional<String> getSetPath()
+                        {
+                            return Optional.empty();
+                        }
+
+                        @Override
+                        public Map<String, String> getSetSessionProperties()
+                        {
+                            return null;
+                        }
+
+                        @Override
+                        public Set<String> getResetSessionProperties()
+                        {
+                            return null;
+                        }
+
+                        @Override
+                        public Map<String, ClientSelectedRole> getSetRoles()
+                        {
+                            return null;
+                        }
+
+                        @Override
+                        public Map<String, String> getAddedPreparedStatements()
+                        {
+                            return null;
+                        }
+
+                        @Override
+                        public Set<String> getDeallocatedPreparedStatements()
+                        {
+                            return null;
```

Comment:
throw `UnsupportedOperationException` on any method that's not implemented

Reply:
done